src/utils.js

```diff
@@ -502,25 +509,20 @@ export default function createUtilsModule({
   };
 
   /**
-   * Convert to fractional monetary unit of Circles
-   * named Freckles.
+   * Convert to fractional monetary unit of Circles named Freckles
    * @namespace core.utils.toFreckles
    * @param {string|number} value - value in Circles
    * @return {string} - value in Freckles
    */
-  const toFreckles = (value) => {
-    return web3.utils.toWei(`${value}`, 'ether');
-  };
+  const toFreckles = (value) => ethers.utils.parseUnits(`${value}`).toString();
 
   /**
-   * Convert from Freckles to Circles number.
+   * Convert from Freckles to Circles number
    * @namespace core.utils.fromFreckles
    * @param {string|number} value - value in Freckles
    * @return {number} - value in Circles
    */
-  const fromFreckles = (value) => {
-    return parseInt(web3.utils.fromWei(`${value}`, 'ether'), 10);
-  };
+  const fromFreckles = (value) => Number(ethers.utils.formatEther(`${value}`));
```

Comment:
The functions before was expected to return a string, so here I see unnecessary to parse to Number, no?
https://web3js.readthedocs.io/en/v1.10.0/web3-utils.html#towei